src/spyglass/common/common_ephys.py

```diff
@@ -132,15 +136,14 @@ def make(self, key):
                     "region_id": BrainRegion.fetch_add(
                         region_name=elect_data.group.location
                     ),
-                    "x": elect_data.x,
```

Comment:
I think you could keep the existing format and just use `get`.

```python
"x": elect_data.get('x'),
```

Does `get` work for each of these? With `a.b` notation, I sometimes have to use `getattr(a, 'b', None)` where `None` must be specified to avoid an error when `a` doesn't have a `b` attr

Reply:
Made this change